pkg/asset/installconfig/openstack/validation/machinepool.go

```diff
@@ -15,39 +15,45 @@ type flavorRequirements struct {
 }
 
 var (
+	minimumStorage          = 25
 	ctrlPlaneFlavorMinimums = flavorRequirements{
 		RAM:   16,
 		VCPUs: 4,
-		Disk:  25,
+		Disk:  minimumStorage,
 	}
 	computeFlavorMinimums = flavorRequirements{
 		RAM:   8,
 		VCPUs: 2,
-		Disk:  25,
+		Disk:  minimumStorage,
 	}
 )
 
 // ValidateMachinePool checks that the specified machine pool is valid.
 func ValidateMachinePool(p *openstack.MachinePool, ci *CloudInfo, controlPlane bool, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
 
+	var checkStorageFlavor bool
```

Comment:
This is both trivial and highly subjective! Writing this only for the thought process because we're both relatively new to golang :) I'm not asking for a change.

checkStorageFlavor is defined to be false here, so the assignment to false on L39 is technically redundant. However, it's explicitness is nice. I might have written:

```
checkStorageFlavor := false
if p.RootVolume != nil {
  ...
} else {
  // Not using root volume, so must check flavor
  checkStorageVolume = true
}
```